config/config.go

```diff
@@ -326,10 +326,9 @@ func NewConfig() (c *Config) {
 				Auth: Auth{
 					Type: AuthTypeNone,
 				},
-				Enabled:           false,
+				Enabled:           true,
 				NamespaceSelector: true,
-				InClusterURL:      "https://tracing.istio-system.svc:16686",
-				URL:               "http://tracing.istio-system:16686",
```

Comment:
Is this URL setting getting removed? If so, we need to remove it in the operator in several places:

(the comment and the setting itself): https://github.com/kiali/kiali/blob/v1.11.0/operator/deploy/kiali/kiali_cr.yaml#L385-L401
(the default): https://github.com/kiali/kiali/blob/v1.11.0/operator/roles/kiali-deploy/defaults/main.yml#L122

Reply:
I may be wrong. Looks like the URL setting is still there: https://github.com/kiali/kiali/blob/56e5ef762f5cc9eb8633fafcd63fc630429eeea9/config/config.go#L129

Looks like we just remove the default value here in config.go?

So resolve this conversation if URL remains in the schema.

Reply:
Yes, URL still exists but we cannot attribute any default value. I could have written `URL: ""` as well.

Reply:
That's fine - I think we are leaving out any default values of empty string so we don't make that structure bigger than it actually is.

We really only need this long structure of defaults for the unit tests - IIRC, this is never used when actually deployed (in upstream istio or maistra) - because the configmap will be used and since the operator puts every value in the configmap, all of these defaults are eventually overridden anyway (again, IIRC)